File: Data Structures & Algorithms/number-of-senior-citizens/submission-1.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        count = 0
        for i in details:
            if int(i[11]) > 6:
                count += 1
            elif int(i[11]) == 6:
                if int(i[12]) == 0:
                    pass
                else:
                    count += 1
        return count 

File: Data Structures & Algorithms/maximum-number-of-balloons/submission-1.java
class Solution {

    public int maxNumberOfBalloons(String text) {
        HashMap<Character, Integer> count= new HashMap<>();
        count.put('b',0);
        count.put('a', 0);
        count.put('l', 0);
        count.put('o', 0);
        count.put('n', 0);
        for (int i = 0; i < text.length(); i++){
            if (count.containsKey(text.charAt(i))){
                if (text.charAt(i) == 'l' || text.charAt(i) == 'o' ){
                    count.put(text.charAt(i), count.get(text.charAt(i)) + 1);
                } else {
                    count.put(text.charAt(i), count.get(text.charAt(i)) + 2);
                }

            }
        }


        int min = count.get('b');
        for (Character c : count.keySet()){
            min = Math.min(min, count.get(c));
        }
        return min / 2;
    }
}